src/components/ui/Chat/ChatBubble/ChatBubbleContainer.tsx

```diff
@@ -47,10 +45,15 @@ export const ChatBubbleContainer: FC<ChatBubbleContainerProps> = React.forwardRe
     const {
       timestamp,
       actions,
+      tooltipContainerId,
       a11yLabel = 'Open channel options',
       ...rest
     } = props;
 
+    const IconButtonWithToolTip = useMemo(
```

Comment:
We are repeating `useMemo` internally in the components specifically for the `WithTooltip(component, tooltipContainerId)`. Should we have `useMemoWithToolTip` as a utility function or more generally memoize component passed to `WithToolTip` component itself?

Reply:
If this was something we would continually reuse, I think it'd be worth it. But since this is a fix until a better solution comes along, I think it's a little overkill. But if you guys prefer, I can split it out

Reply:
Yeah I agree, seems like an overkill as anyway this is going to be re-worked. Np.